src/main/java/org/dataone/hashstore/hashfs/HashUtil.java

```diff
@@ -88,26 +95,67 @@ public String shard(int depth, int width, String digest) {
                 stringArray.add(str);
             }
         }
-        String shardedPath = "/" + String.join("/", stringArray);
-        return shardedPath;
+        String stringShard = "/" + String.join("/", stringArray);
```

Comment:
It seems like compatibility issues continue to be a challenge. I added "/" in the beginning here so that the code where I call it would look cleaner - but this makes the method messy, so I removed "/" here and appended it where applicable in HashFileStore (ex. `this.objectStoreDirectory.toString() + "/" + objShardString;`)